pkg/cache/unavailableofferings.go

```diff
@@ -48,13 +48,17 @@ func NewUnavailableOfferings() *UnavailableOfferings {
 }
 
 // IsUnavailable returns true if the offering appears in the cache
-func (u *UnavailableOfferings) IsUnavailable(instanceType ec2types.InstanceType, zone, capacityType string) bool {
+func (u *UnavailableOfferings) IsUnavailable(instanceType string, zone, capacityType string) bool {
 	_, found := u.cache.Get(u.key(instanceType, zone, capacityType))
 	return found
 }
 
+func (u *UnavailableOfferings) IsReservationUnavailable(reservationID string) bool {
```

Comment:
What's the utility of this now? Is this intended to be a TODO that we will come back to or is there a use-case for calling this function but always returning false?

Reply:
Dropped this in favor of tracking reservation availability in the CapacityReservationProvider.